camp/JHall/leftJoystickToTalon/src/main/java/frc/robot/commands/changeSpeed.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands;
```

Comment:
the name of this file should be ChangeSpeed.java to match the class name with the same casing.